java-generator/it/src/it/cert-manager/expected/Auth.expected

```diff
@@ -82,4 +82,9 @@ public class Auth implements io.fabric8.kubernetes.api.model.KubernetesResource
     public void setAdditionalProperties(java.util.Map<String, Object> additionalProperties) {
         this.additionalProperties = additionalProperties;
     }
+
+    @com.fasterxml.jackson.annotation.JsonAnySetter()
+    public void setAdditionalProperties(String key, Object value) {
```

Comment:
Should this be setAdditionalProperty singular?

Reply:
Surprisingly not, the integration test fails if I make it singular.

Reply:
So probably some jackson magic :)

Reply:
yup, at least now is properly tested 🙂